test/Conversion/TorchOnnxToTorch/op_wise_domain.mlir

```diff
@@ -0,0 +1,8 @@
+// RUN: torch-mlir-opt <%s --split-input-file -convert-torch-onnx-to-torch | FileCheck %s
+
+// CHECK-LABEL: @test_quantizelinear_si8
+func.func @test_quantizelinear_si8(%arg0: !torch.vtensor<[6],f32>, %arg1: !torch.vtensor<[],f32>, %arg2: !torch.vtensor<[],si8>) -> !torch.vtensor<[6],si8> attributes {torch.onnx_meta.ir_version = 9 : si64, torch.onnx_meta.opset_version = 16 : si64} {
```

Comment:
This test would have previously caused the error that you removed?

Reply:
No: as far as I have seen, only one version of each supported op is currently accepted by torch-mlir (I might be wrong but checking for both multiple occurrences of the same op name as well as `OpNameV[0-9]+` returns no results). So, we can't test right now that a specific version of the op is indeed used.

However, now that you say it, I should add a test for an unsupported op due to a too early version. Let me try and come up with this :)